frontend/admin/src/js/components/user/UserTable.tsx

```diff
@@ -96,13 +83,43 @@ export const UserTable: React.FC = () => {
     col: string | undefined;
   }>();
 
+  // merge search bar input with fancy filter state
+  const filterCombiner = (
```

Comment:
re: starting with verb, maybe something descriptive like

```suggestion
      where: buildUserFilterInput(fancyFilter, searchState?.term, searchState?.col),
```

Reply:
sounds reasonable 
https://github.com/GCTC-NTGC/gc-digital-talent/pull/3893/commits/a2cb77df9d066a66a0701d8289082a4d0f68742f